src/include/gcc.h

```diff
@@ -128,6 +128,16 @@ __wt_atomic_cas_ptr(void *vp, void *old, void *newv)
     return (WT_ATOMIC_CAS((void **)vp, &old, newv));
 }
 
+/*
+ * __wt_atomic_load_acquire64 --
+ *     Read a with atomic acquire synchronization.
+ */
+static inline void
+__wt_atomic_load_acquire64(void *ret, void *ptr)
```

Comment:
It makes me nervous that the order of parameters is swapped between calls into this function and the subsequent call. Especially since there isn't any meaningful type checking. Does that approach make this much more obvious in the code? If so, it should be very clearly documented, I think.

Reply:
I swap the order just to make it consistent to `WT_ORDERED_READ`. I'll think about if there are better ways.